src/toolbox/methods_category.ts

```diff
@@ -99,6 +99,13 @@ class MethodsCategory {
             methodNamesAlreadyOverridden, contents);
         break;
       case storageModule.ModuleType.OPMODE:
+        const hasSteps = editor.isStepsInWorkspace();
+        if (!hasSteps) {
+          contents.push({
```

Comment:
Please add a method in mrc_steps.ts that creates the block and call it here.

Also, the block should have extraState.

Reply:
This is fixed in https://github.com/alan412/systemcore-blocks-interface/pull/17